<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1451</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="Pressure" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>580</width>
      <height>240</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>580</width>
      <height>240</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="Altitude" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>580</width>
      <height>240</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>240</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="Temperature" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>481</width>
      <height>240</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>240</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="RawData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>741</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="Gyro" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>560</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="Gyro_Z" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1451</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionCalibrate"/>
   </widget>
   <addaction name="menuStart"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
